pkg/helper/machinepools/helpers.go

```diff
@@ -14,14 +14,20 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation"
 )
 
-func MinNodePoolReplicaValidator() interactive.Validator {
+func MinNodePoolReplicaValidator(autoscaling bool) interactive.Validator {
 	return func(val interface{}) error {
 		minReplicas, err := strconv.Atoi(fmt.Sprintf("%v", val))
 		if err != nil {
 			return err
 		}
-		if minReplicas < 1 {
-			return fmt.Errorf("min-replicas must be greater than zero")
+		if autoscaling {
+			if minReplicas < 1 {
+				return fmt.Errorf("min-replicas must be greater than zero")
```

Comment:
do we have such kind of messages? I would have expected something expanded 
```suggestion
				return fmt.Errorf("The minimum number of replicas must be greater than zero")
```

Reply:
Note that I've reused similar wording for same validation in the classic machine pools.

I.e.
![image](https://github.com/openshift/rosa/assets/1662329/7ecd9869-4428-4881-92b1-0728694a0db4)

Where the message focuses on the field first "replicas" for non autoscaling vs "min-replicas" for autoscaling.

Reply:
ok, let's be consistent then